routes/index.js

```diff
@@ -585,16 +524,6 @@ function checkRegistration(req, res, next) {
     for (let val in mappedErrors) {
       req.flash('error', mappedErrors[val]['msg']);
     }
-
-    const allData = matchedData(req);
```

Comment:
Why is this removed? Without it the form will not save fields when there is an error. Also I am pretty sure this does not affect message rendering

Reply:
It definitely wasn't intentional. I must have missed this when I was copy/pasting the stuff that I didn't get from pulling master when I was out of sync. Adding back and resubmitting.